### PR TITLE
Changed "Gevoelstemperatuur" to "Voelt als". This is shorter and save…

### PR DESCRIPTION
**Pull request by Travelbacon**
Changed the Dutch text "Gevoelstemperatuur"  to "Voelt als". 

Texts on the Mirror should be short and punctual as possible. It is quicker to read and saves mirror space. Shorter texts will also align better with the modules. In the weather module the text "Voelt als"  matches better in length with the other lines of information.

### CODE REVIEW

**Comment by MichMich** [comment]
Hmm, this is a difficult PR. I understand your thought. But _gevoelstemperatuur_ is the official term used by the KNMI. Maybe it's a good idea to open a POLL on the forum about this, to see what the community prefers.

**Comment by Travelbacon** [comment]
What is the best place to hold this poll?

**Comment by MichMich** [comment]
I suggest doing it on the forum. http://forum.magicmirror.builders

**Comment by Travelbacon** [comment]
Done. I took the liberty to end it end of august. This will give it plenty of time with a proper deadline
https://forum.magicmirror.builders/topic/12981/poll-change-gevoelstemperatuur-to-voelt-als-for-a-better-allignment

**Comment by MichMich** [comment]
Closed due to inactivity. Feel free to reopen.

**Comment by Travelbacon** [comment]
I would say to change it. Seeing the low amount of people who cares to change it.
9 people voted. 7 Yay, 2 Nay
https://forum.magicmirror.builders/topic/12981/poll-change-gevoelstemperatuur-to-voelt-als-for-a-better-allignment

**Comment by MichMich** [comment]
It seems to have merge conflicts. If you can fix them I'll merge it (I'm in the phone so I can't fix it.)

**Comment by Travelbacon** [comment]
Sure.

**Comment by Travelbacon** [comment]
I think it should be ready for merge now. I mainly work with GIT as it is CVS. ;)

File: translations/nl.json [hunk 1]
```diff
@@ -28,6 +28,11 @@
 	"UPDATE_NOTIFICATION_MODULE": "Update beschikbaar voor {MODULE_NAME} module.",
 	"UPDATE_INFO_SINGLE": "De huidige installatie loopt {COMMIT_COUNT} commit achter op de {BRANCH_NAME} branch.",
 	"UPDATE_INFO_MULTIPLE": "De huidige installatie loopt {COMMIT_COUNT} commits achter op de {BRANCH_NAME} branch.",
+<<<<<<< HEAD
```
Comment (rejas) [inline translations/nl.json:31]: Please remove those GIT lines indicating where the merge conflict happened @Travelbacon

**Pull request merged**